docs/cli/cli.md

```diff
@@ -0,0 +1,122 @@
+# 💻 CLI for `jstz`
+
+This guide will instruct through how to use the command line interface for `jstz`.
+
+## Overview
+
+`jstz` provides a simple command line interface where the user can easily run the sandbox environment and test smart functions.
+
+## Basic Commands
+
+- `jstz <COMMAND>`: The primary way to interact with `jstz`.
+- `help`: Displays a general help message or help information for specific subcommands.
+- `--version (-V)`: Displays the current version of `jstz`.
```

Comment:
I think what you're mainly conveying here is flags (e.g. `--help` or perhaps later on `--config`, etc). In this case, I think using a custom container to make a note of these would look best (https://vitepress.dev/guide/markdown#custom-containers)